Replace babel with SWC & minor changes in `getting started` (#34282)
vercel/next.js: docs/getting-started.md

Welcome to the Next.js documentation!

> If you're new to Next.js, we recommend that you start with the [learn course](https://nextjs.org/learn/basics/create-nextjs-app).

Reviewer: ```suggestion
If you're new to Next.js, we recommend starting with the [learn course](https://nextjs.org/learn/basics/create-nextjs-app).
```

So far, we get:

> - Automatic compilation and bundling (with webpack and [SWC](https://nextjs.org/blog/next-12#faster-builds-and-fast-refresh-with-rust-compiler))

Reviewer: ```suggestion
- Automatic compilation and [bundling](/docs/advanced-features/compiler.md)
```

<div class="card">
  <a href="/docs/basic-features/built-in-css-support.md">
    <b>CSS Support:</b>

>     <small>Learn more about how to use the built-in CSS support to add custom styles to your app.</small>

Reviewer: ```suggestion
    <small>Built-in CSS support to add custom styles to your app.</small>
```

Reviewer: "Learn more about how to use" could make this a bit long
<img width="843" alt="CleanShot 2022-02-12 at 20 56 24@2x" src="https://user-images.githubusercontent.com/9113740/153736322-135e3259-c8c4-4aec-9ed3-82d005e69176.png">
